schemas/devContainerFeature.schema.json

```diff
@@ -0,0 +1,220 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Development Container Feature Metadata",
+  "description": "Development Container Features Metadata (devcontainer-feature.json). See https://containers.dev/implementors/features/ for more information.",
+    "definitions": {
+    "Feature": {
+      "additionalProperties": false,
+      "properties": {
+        "capAdd": {
+					"description": "Passes docker capabilities to include when creating the dev container.",
+					"examples": [ "SYS_PTRACE" ],
+          "items": {
+            "type": "string"
+          },
+          "type": "array"
+        },
+        "containerEnv": {
+          "description": "Container environment variables.",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "type": "object"
+        },
+        "customizations": {
+          "description": "Tool-specific configuration. Each tool should use a JSON object subproperty with a unique name to group its customizations.",
+          "additionalProperties": true,
+          "type": "object"
+        },
+        "description": {
+          "description": "Description of the Feature. For the best appearance in an implementing tool, refrain from including markdown or HTML in the description.",
+          "type": "string"
+        },
+        "documentationURL": {
+          "description": "URL to documentation for the Feature.",
+          "type": "string"
+        },
+        "entrypoint": {
+          "description": "Entrypoint script that should fire at container start up.",
+          "type": "string"
+        },
+        "id": {
+          "description": "ID of the Feature. The id should be unique in the context of the repository/published package where the feature exists and must match the name of the directory where the devcontainer-feature.json resides.",
+          "type": "string"
+        },
+        "init": {
+          "description": "Adds the tiny init process to the container (--init) when the Feature is used.",
+          "type": "boolean"
+        },
+        "installsAfter": {
+          "description": "Array of ID’s of Features that should execute before this one. Allows control for feature authors on soft dependencies between different Features.",
+          "items": {
+            "type": "string"
+          },
+          "type": "array"
+        },
+        "licenseURL": {
+          "description": "URL to the license for the Feature.",
+          "type": "string"
+        },
+        "mounts": {
```

Comment:
Also add a description for "mounts". The one from "Mounts" isn't shown in IntelliSense / hovers.